extensions/ql-vscode/src/data-extensions-editor/queries/index.ts

```diff
@@ -0,0 +1,6 @@
+import { fetchExternalApisQuery as javaFetchExternalApisQuery } from "./java";
+import { Query } from "./query";
+
+export const fetchExternalApiQueries: Record<string, Query> = {
```

Comment:
We do, but the `DatabaseItem.language` is a `string`, so we'd need to cast it to a `QueryLanguage` which might hide problems rather than actually giving us type safety. I've changed it to use a `QueryLanguage` though since that will prevent us from adding unsupported types.